.requirements

```diff
@@ -8,4 +8,4 @@ RESTY_OPENSSL_VERSION=1.1.1j
 RESTY_PCRE_VERSION=8.44
 LIBYAML_VERSION=0.2.5
 KONG_GO_PLUGINSERVER_VERSION=v0.6.1
-KONG_BUILD_TOOLS_VERSION=4.13.0
+KONG_BUILD_TOOLS_VERSION=37ece0b806501ba85007cd0efba117e58a6fb78f
```

Comment:
note: this is a commit SHA that points to this required kong-build-tools patch: https://github.com/Kong/kong-build-tools/commit/37ece0b806501ba85007cd0efba117e58a6fb78f

todo: merge https://github.com/Kong/kong-build-tools/pull/378, bump kong-build-tools and pin the new version here

```suggestion
KONG_BUILD_TOOLS_VERSION=4.14.0
```